### PR TITLE
add support for SSL connection in log4net

### PR DESCRIPTION
**Pull request by energister**
plus corresponding changes in log4net example

### CODE REVIEW

File: Source/EasyGelf.Core/Transports/Tcp/TcpTransportConfiguration.cs [hunk 1]
```diff
@@ -1,9 +1,48 @@
...
+        {
+            if (host != null)
+            {
+                return host;
+            }
+
+            var remoteIpAddress = Dns.GetHostAddresses(RemoteAddress)
+                .Shuffle()
+                .FirstOrDefault() ?? IPAddress.Loopback;
+
```
Comment (Pliner) [inline Source/EasyGelf.Core/Transports/Tcp/TcpTransportConfiguration.cs]: Could you replace it to .DefaultIfEmpty(IPAddress.Loopback).First()?

Comment (energister) [inline Source/EasyGelf.Core/Transports/Tcp/TcpTransportConfiguration.cs]: done

**Comment by Pliner** [comment]
I hope you tested these changes 😜

**Comment by energister** [comment]
After your question I've tested last code review changes. Everything works as expected. So relax :)

**Pull request merged**